### DISCUSSION TITLE
Search String for Collection Results

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#21053] [Ideas]
Would be nice to have the user able to search for specific beatmaps in a collection, furthermore a set of collections

Ex. ("in collection1 OR collection2 star>5")

**Comment by nagi-desuuu** [comment]
You can already do that via the collections drop-down, although what you're suggesting may be a good convenience factor.

https://user-images.githubusercontent.com/75596775/199198396-195137ad-2a0b-4152-8d47-55ba11ce9579.mp4

**Reply by 19peytonsmith** [reply]
Oh nice, this won't work with multiple collections though right?
